<?xml version="1.0"?>
<launch>
  <!-- We resume the logic in empty_world.launch, changing only the name of the world to be launched -->
  <include file="$(find gazebo_ros)/launch/empty_world.launch">
  </include>

  <!-- Spawn simulated quadrotor uav -->

   <group ns="ground">
    <include file="$(find quad_simulator)/launch/ground.launch">
      <arg name="robot_name" value="ground"/>
      <arg name="init_pose" value="-x 0 -y -0 -z 0"/>
    </include>
  </group>
   
   <group ns="robot0">
    <include file="$(find quad_simulator)/launch/one_patrolbot.launch">
      <arg name="robot_name" value="robot0"/>
      <arg name="init_pose" value="-x 5 -y 0 -z 0 -Y -1.57"/>
    </include>
  </group>

  
  <group ns="robot1">
    <include file="$(find quad_simulator)/launch/one_patrolbot.launch">
      <arg name="robot_name" value="robot1"/>
      <arg name="init_pose" value="-x 0 -y 5 -z 0 -Y 0"/>
    </include>
  </group>
  <group ns="robot2">
    <include file="$(find quad_simulator)/launch/one_patrolbot.launch">
      <arg name="robot_name" value="robot2"/>
      <arg name="init_pose" value="-x -5 -y 0 -z 0 -Y 1.57"/>
    </include>
  </group>
  <group ns="robot3">
    <include file="$(find quad_simulator)/launch/one_patrolbot.launch">
      <arg name="robot_name" value="robot3"/>
      <arg name="init_pose" value="-x 0 -y -5 -z 0 -Y 0"/>
    </include>
  </group>

 <!-- Spawn Albatross 
  <include file="$(find hector_quadrotor_gazebo)/launch/spawn_quadrotor.launch" >
    <arg name="model" value="$(find quad_simulator)/urdf/quadrotor.gazebo.xacro"/>
  </include>
-->
<!-- Spawn Hector Quadrotor -->
  <include file="$(find hector_quadrotor_gazebo)/launch/spawn_quadrotor.launch" >
    <arg name="model" value="$(find quad_simulator)/urdf/quadrotor_downward_cam.gazebo.xacro"/>
  </include>


  
  <group ns="robot4">
    <include file="$(find quad_simulator)/launch/one_robot.launch">
      <arg name="robot_name" value="robot4"/>
      <arg name="init_pose" value="-x 1 -y 0 -z 0 -Y 0"/>
    </include>
   </group>

  
  <group ns="robot5">
    <include file="$(find quad_simulator)/launch/one_robot.launch">
      <arg name="robot_name" value="robot5"/>
      <arg name="init_pose" value="-x 0.807 -y 0.657 -z 0 -Y 0.683"/>
    </include>
  </group>

  
  <group ns="robot6">
    <include file="$(find quad_simulator)/launch/one_robot.launch">
      <arg name="robot_name" value="robot6"/>
      <arg name="init_pose" value="-x 0.35 -y 1 -z 0 -Y 1.234"/>
    </include>
  </group>
  <group ns="robot7">
    <include file="$(find quad_simulator)/launch/one_robot.launch">
      <arg name="robot_name" value="robot7"/>
      <arg name="init_pose" value="-x -0.35 -y 1 -z 0 -Y 1.907"/>
    </include>
  </group>
  <group ns="robot8">
    <include file="$(find quad_simulator)/launch/one_robot.launch">
      <arg name="robot_name" value="robot8"/>
      <arg name="init_pose" value="-x -0.807 -y 0.657 -z 0 -Y 2.458"/>
    </include>
  </group>
  <group ns="robot9">
    <include file="$(find quad_simulator)/launch/one_robot.launch">
      <arg name="robot_name" value="robot9"/>
      <arg name="init_pose" value="-x -1 -y 0 -z 0 -Y 0"/>
    </include>
  </group>
  <group ns="robot10">
    <include file="$(find quad_simulator)/launch/one_robot.launch">
      <arg name="robot_name" value="robot10"/>
      <arg name="init_pose" value="-x -0.807 -y -0.657 -z 0 -Y -2.458"/>
    </include>
  </group>
  <group ns="robot11">
    <include file="$(find quad_simulator)/launch/one_robot.launch">
      <arg name="robot_name" value="robot11"/>
      <arg name="init_pose" value="-x -0.35 -y -1 -z 0 -Y -1.907"/>
    </include>
  </group> 
<group ns="robot12">
    <include file="$(find quad_simulator)/launch/one_robot.launch">
      <arg name="robot_name" value="robot12"/>
      <arg name="init_pose" value="-x 0.35 -y -1 -z 0 -Y -1.234"/>
    </include>
  </group> 

<group ns="robot13">
    <include file="$(find quad_simulator)/launch/one_robot.launch">
      <arg name="robot_name" value="robot13"/>
      <arg name="init_pose" value="-x 0.807 -y -0.657 -z 0 -Y -0.683"/>
    </include>
  </group> 

</launch>
